simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/configuration/SimulatorConfiguration.java

```diff
@@ -42,6 +46,24 @@
         private int connectors;
     }
 
+    @Data
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class MeterValuesConfiguration {
+        /**
+         * How often send meter values in milliseconds
+         */
+        @Builder.Default
+        private long meterValuesIntervalMs = DEFAULT_METER_VALUES_INTERVAL;
+
+        /**
+         * Power consumed for each power consumption interval
+         */
+        @Builder.Default
+        private long powerConsumptionPerInterval = DEFAULT_POWER_CONSUMPTION_PER_INTERVAL;
```

Comment:
Since we don't specify the unit - it will be watt-hour (see 1.52. UnitOfMeasureType in spec), so let's rename it to 'consumptionWattHour' and when sending we can increment by `(sendMeterValuesIntervalSec X consumptionWattHour) / 3600` with rounding.

Then we'll be sending not some abstract units, but data that's clear for everyone